### PR TITLE
RHIDP-3681-Remove-Incorrect-Links-In-Documentation

### PR DESCRIPTION
**Pull request by jmagak**
**IMPORTANT: Do Not Merge - To be merged by Docs Team Only**

**Version(s):** 1.2 

**Issue:** https://issues.redhat.com/browse/RHIDP-3681

**Link to docs preview:**

**Reviews:**
- [x] SME: @nickboldt 
- [ ] QE: @ mention assignee
- [x] Docs review: @themr0c 
- [ ] Additional review: @mention assignee (by writer)

**Additional information:**

### CODE REVIEW

File: assemblies/assembly-admin-templates.adoc [hunk 1]
```diff
@@ -23,4 +23,4 @@ include::modules/templates/proc-adding-templates.adoc[leveloffset=+1]
 [role="_additional-resources"]
 .Additional resources
 * link:{LinkGettingStartedGuide}[Configuring a GitHub App in {product-short}]
-* link:{LinkGettingStartedGuide}[Enabling the GitLab OAuth authentication provider]
+* link:{LinkGettingStartedGuide}[Configuring GitHub authentication]
```
Comment (gwynnemonahan) [inline assemblies/assembly-admin-templates.adoc:26]: Are we just linking to the Getting Started Guide, or are we linking to the "Configuring GitHub authentication" section of the Getting Started Guide? 

This https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/getting_started_with_red_hat_developer_hub/ref-rhdh-supported-configs_rhdh-getting-started#configuring-github-authentication sends the user to the "Configuring GitHub authentication" section. 

{LinkGettingStartedGuide} sends users here https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/getting_started_with_red_hat_developer_hub/index and then they have to try and find their way to "Configuring GitHub authentication." They have to know to navigate to "Supported Configurations" > "Adding source control"

I'm not familiar with how linking works for RHDH docs, so I don't know if {LinkGettingStartedGuide} is all that can be done and users have to navigate on their own, or if we can direct users specifically to "Configuring GitHub authentication."

Comment (jmagak) [reply assemblies/assembly-admin-templates.adoc:26]: The link that should be here is https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/getting_started_with_red_hat_developer_hub/ref-rhdh-supported-configs_rhdh-getting-started#configuring-github-authentication that leads users specifically to the 'Configuring GitHub authentication' section. 
I am not quite sure how this is done in RHDH as well. I am looking into it though.

Comment (nickboldt) [reply assemblies/assembly-admin-templates.adoc:26]: given 1.3 is a week away, should we be focusing on 1.2 docs at this time? 

Here's an example of how to link to a book and then an anchor with the book: 

https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/525/files#diff-f2d027fe6bc6168c7119ae3c44f1ca07f0a261b1c3ffa9a7e4296f2c75c95ac2R66

For this to work you also need an id defined for the section:

https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/525/files#diff-24ade8b6ecf1a6c361c4ebcfb3727ca26b9f7e013822e602001d38e1e900dbadR50

Comment (jmagak) [reply assemblies/assembly-admin-templates.adoc]: Thank you @nickboldt for your feedback. I have corrected the link. 

The team is currently working on 1.3 and as a new member of the team, I am assisting in handling these tasks same as those of 1.2. Also, this PR will be cherry-picked to `1.2.x` as well as `1.3.x.`

**Review by themr0c — APPROVED** [review]

**Pull request merged**